src/Logging/composer.json

```diff
@@ -0,0 +1,22 @@
+{
+    "name": "google/cloud-logging",
+    "description": "Stackdriver Logging Client for PHP",
+    "license": "Apache-2.0",
+    "minimum-stability": "stable",
+    "require": {
+        "google/cloud-core": "*"
+    },
```

Comment:
For logging/pubsub does it make sense to add in a suggested field here with the gax/proto libs?